packages/dds/tree/src/tableSchema.ts

```diff
@@ -774,30 +781,33 @@ export namespace System_TableSchema {
 				return column;
 			}
 
-			public removeAllColumns(): ColumnValueType[] {
-				// TypeScript is unable to narrow the row type correctly here, hence the cast.
-				// See: https://github.com/microsoft/TypeScript/issues/52144
-				return this.removeColumns(this.columns as unknown as ColumnValueType[]);
-			}
+			public removeRows(
+				indexOrRows: number | undefined | readonly string[] | readonly RowValueType[],
+				count?: number | undefined,
```

Comment:
Just to match `removeColumns()`.
```suggestion
				count: number | undefined = undefined,
```

Reply:
Ah, had this set to autocomplete. I'll make a separate PR to update this. Good call!